commands/trigger.go

```diff
@@ -75,7 +77,7 @@ func feedParameters(feed string, lifecycle string, trigger *QualifiedName, authK
  *
  * NOTE: this function will exit in case of a processing error since it indicates a problem parsing parameters.
  */
-func createOrUpdate(fqn *QualifiedName, trigger *whisk.Trigger, overwrite bool) {
+func createOrUpdate(Client *whisk.Client, fqn *QualifiedName, trigger *whisk.Trigger, overwrite bool) {
```

Comment:
Now that `Client` is a parameter, can we delete the comment `// TODO get rid of these global modifiers`?

Reply:
yes, now it's "local".